composer.json

```diff
@@ -32,5 +33,9 @@
         "branch-alias": {
             "dev-master": "1.x-dev"
         }
+    },
+    "scripts": {
+        "cs": "php-cs-fixer fix --verbose --diff",
+        "test": "phpunit"
```

Comment:
@alcohol 

Worth adding a `.github/CONTRIBUTING.md` with notes about this, and referencing it from `README.md`?